### DISCUSSION TITLE
İBAX.İO BELGELER

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#635] [Ideas] [answered]
![Screenshot_2022-01-25-00-34-04-68](https://user-images.githubusercontent.com/96292241/150869730-fe59fd34-f156-4556-9977-faeb3982c441.jpg)
![Screenshot_2022-01-25-00-33-47-74](https://user-images.githubusercontent.com/96292241/150869737-04ad8a0a-072d-45e1-b21a-05e88051b9ff.jpg)
[Buraya giderek ibax dokümanları hakkında daha fazla bilgi sahibi  @olabilirsiniz](https://github.com/Akrepkapo/documentation.git) 
   @ibaxio @BadAssKyle

**Comment by ibaxio** [comment, marked as answer]
You could visit https://docs.ibax.io to research Technical Documentation.

We will appreciate you if you translate it to Turkish.

**Reply by Akrepkapo** [reply]
Teşekkürler 🤗🤗🤗
